modules/core/include/tiovx_modules_cbs.h

```diff
@@ -81,15 +81,19 @@
 #include "tiovx_pixelwise_add_module.h"
 #include "tiovx_lut_module.h"
 
-#if defined(SOC_J721E) || defined(SOC_J721S2) || defined(SOC_J784S4) || defined(SOC_J722S) || defined(SOC_J742S2) 
+#if defined(SOC_J721E) || defined(SOC_J721S2) || defined(SOC_J784S4) || defined(SOC_J722S) || defined(SOC_J742S2) || defined(SOC_AM62A)
+#if !defined(SOC_AM62A)
 #include "tiovx_display_module.h"
+#endif
 #include "tiovx_capture_module.h"
 #include "tiovx_aewb_module.h"
+#if !defined(SOC_AM62A)
```

Comment:
are these exluded header files needed for AM62A-Linux ? if yes, please add QNX flag as well